### PR TITLE
Cleanup around the new IAddImport service.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@dpoeschl Ths is ready for you.

File: src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs [hunk 1]
```diff
@@ -559,9 +534,16 @@ private NameSyntax CreateNameSyntax(IReadOnlyList<string> namespaceParts, int in
...
             SyntaxNode contextNode)
         {
-            var addImportService = document.GetLanguageService<IAddImportService>();
+            var addImportService = document.GetLanguageService<IAddImportsService>();
 
             var nameSyntax = namespaceOrTypeSymbol.GenerateNameSyntax();
+
+            // We need to create our using in two passes.  This is because we need a using
+            // directive so we can figure out where to put it.  Then, once we figure out
+            // where to put it, we might need to change it a bit (i.e. removing 'global'
```
Comment (dpoeschl) [inline src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs]: `i.e.` => `e.g.,`

File: src/Workspaces/VisualBasic/Portable/AddImports/VisualBasicAddImportsService.vb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    Friend Class VisualBasicAddImportsService
+        Inherits AbstractAddImportsService(Of
+            CompilationUnitSyntax,
+            NamespaceBlockSyntax,
+            ImportsStatementSyntax,
+            ImportsStatementSyntax)
+
+        Protected Overrides Function GetAlias(usingOrAlias As ImportsStatementSyntax) As SyntaxNode
+            Return usingOrAlias.ImportsClauses.OfType(Of SimpleImportsClauseSyntax).
+                                               FirstOrDefault()?.Alias
```
Comment (dpoeschl) [inline src/Workspaces/VisualBasic/Portable/AddImports/VisualBasicAddImportsService.vb]: What about when there's multiple ImportClauses and a later one is an Alias? Or multiple? (I don't know if those are valid cases in VB, but the FirstOrDefault() feels sketchy)

Comment (CyrusNajmabadi) [inline src/Workspaces/VisualBasic/Portable/AddImports/VisualBasicAddImportsService.vb]: Fixed.

**Comment by CyrusNajmabadi** [comment]
Merging as feedback was reponded to.

**Pull request merged**